sourcegraph/about
# Add script to check Node version and install pinned dependencies

Closes #5133 by ensuring that we have some means of guaranteeing that everybody's on the pinned version of Node for this project and installing the pinned dependencies.

## To Test
- Check out the branch and remove your `node_modules` in `website`
- Change your version of Node, run the install script `./script/install.sh`, make sure it exits.
- Set the `YARN` var in the script to something you don't have installed, make sure it exits.
- Ensure your dependencies install correctly.
